packages/iota/src/validation/essencePayload/essencePayload.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright 2020 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+import { ITaggedDataPayload, TAGGED_DATA_PAYLOAD_TYPE } from "../../models/payloads/ITaggedDataPayload";
+import { validateTaggedDataPayload } from "../payloads/payloads";
+import { IValidationResult, failValidation } from "../result";
+
+/**
+ * Validate transaction essence payload.
+ * @param payload The tagged data payload.
+ * @returns The validation result.
+ */
+export function validateEssencePayload(payload: ITaggedDataPayload): IValidationResult {
```

Comment:
Not much point in a new file for this couple of lines, also no `essencePayload` in `binary` folder as it is not a thing really.
Move this function inside `transactionEssence` and just use it.